testonly/hammer/hammer.go

```diff
@@ -285,6 +285,82 @@ func newWorker(cfg *MapConfig, prng *rand.Rand) *mapWorker {
 	}
 }
 
+func (w *mapWorker) retryOneOp(ctx context.Context, s *hammerState) (err error) {
```

Comment:
Just to double check: Have `retry[One]Op` methods been moved without modification here?

Reply:
Yup, they are just a move. I did them in a second commit so that would be clearer, but unfortunately the diff tool here isn't smart enough to show a reordering move clearly.